test/e2e/page-objects/flows/install-test-snap.flow.ts

```diff
@@ -11,27 +11,37 @@ import { WINDOW_TITLES } from '../../helpers';
  *
  * @param driver - WebDriver instance used to interact with the browser.
  * @param buttonName - The name of the button to click.
- * @param withWarning - Whether the installation will have a warning dialog, default is false.
+ * @param options - Optional parameters.
+ * @param options.withWarning - Whether the installation will have a warning dialog, default is false.
+ * @param options.withExtraScreen - Whether there is an extra screen after the Ok, defaults to false.
  */
 export async function openTestSnapClickButtonAndInstall(
   driver: Driver,
   buttonName: keyof typeof buttonLocator,
-  withWarning = false,
+  options: {
+    withWarning?: boolean;
+    withExtraScreen?: boolean;
+  } = {},
```

Comment:
added the withExtraScreen option, so we can know whether to wait for the dialog to close or not. This will mitigate race conditions in all snap specs that install a snap.
Added this as an options object, to imporve readability (see specs below)